core/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -287,6 +287,14 @@ databaseChangeLog:
   - include:
       file: changesets/2024_07_03_family_change_records.yaml
       relativeToChangelogFile: true
+  - include:
+      file: changesets/2024_06_18_admin_task_refactor.yaml
+      relativeToChangelogFile: true
+  - include:
+      file: changesets/2024_06_19_admin_tasks_to_participant_tasks.yaml
+      relativeToChangelogFile: true
```

Comment:
just for my own understanding: are we gonna go back and delete the table in the future? is this just for safety in case the migration fails?

Reply:
yeah exactly, just keeping it for safety. my general thinking is that there's no rush to fully destroy data during a migration because it's cheap to keep around, but once we're sure that everything looks fine post-release, we can go back and delete it